manual/core/request_tracker/README.md

```diff
@@ -77,20 +77,26 @@ datastax-java-driver.advanced.request-tracker {
 All requests are logged under the category
 `com.datastax.oss.driver.internal.core.tracker.RequestLogger`.
 
+Prefix of the log will always contain at least: 
+
+``s0|274426173``
```

Comment:
Formatting is a bit off, the next sentence starts on the same line ([github rendering](https://github.com/datastax/java-driver/tree/java2278/manual/core/request_tracker)).

If you want this to appear as a code block, use triple backticks:

    ```
    s0|274426173
    ```